lxd/patches.go

```diff
@@ -1393,154 +1394,95 @@ func patchRemoveCandidRBACConfigKeys(_ string, d *Daemon) error {
 
 // patchStorageSetVolumeUUID sets a unique volatile.uuid field for each volume and its snapshots.
 func patchStorageSetVolumeUUID(_ string, d *Daemon) error {
```

Comment:
please can you break this into 2 commits, one to disable the old patch and one to add the new one as otherwise its tricky to review.

Reply:
ignore me i was looking at wrong view